inspire_matcher/core.py

```diff
@@ -241,3 +244,12 @@ def _compile_nested_prefix(query, record):
 
 def _get_common_path(paths):
     return '.'.join(os.path.commonprefix([path.split('.') for path in paths]))
+
+
+def _compile_authors_query(query, record):
+    parsed_name = ParsedName(record.get("full_name"))
```

Comment:
`full_name` is required by schema, so you can do
```suggestion
    parsed_name = ParsedName(record["full_name"])
```